File: app/src/main/java/me/manabu/activities/parents/BasicNavigationDrawerActivity.java
package me.manabu.activities.parents;

import com.mikepenz.materialdrawer.Drawer;

import me.manabu.utils.NavigationDrawer;

abstract public class BasicNavigationDrawerActivity extends BasicToolbarActivity {

    protected Drawer drawer;

    protected void initDrawer() {
        drawer = new NavigationDrawer(this, toolbar).getDrawer();
    }

    public Drawer getDrawer() {
        return drawer;
    }
}

File: app/src/main/java/me/manabu/activities/parents/BasicNavigationDrawerActivity.kt
package me.manabu.activities.parents

import com.mikepenz.materialdrawer.Drawer

import me.manabu.utils.NavigationDrawer

abstract class BasicNavigationDrawerActivity : BasicToolbarActivity() {

    var drawer: Drawer? = null
        private set

    protected fun initDrawer() {
        drawer = NavigationDrawer(this, toolbar).drawer
    }
}